index.html

```diff
@@ -44,10 +44,10 @@ <h1>Tic Tac Toe</h1>
     <td><input type="checkbox" /></td>
```

Comment:
This is a test suggestion.
```suggestion
    <td><input type="checkbox" /></td>
```